content/en/integrations/fluentbit.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Fluent Bit
+name: fluentbit
+kind: integration
+description: "Configure Fluent Bit to collect, parse, and forward log data from several different sources to Datadog for monitoring."
+short_description: "Collect, parse, and forward log data from several different sources to Datadog for monitoring."
+categories:
+- log collection
+doc_link: /integrations/fluentbit/
+has_logo: true
+integration_title: Fluent Bit
+is_public: true
+public_title: Datadog-Fluent Bit Integration
+further_reading:
+- link: "https://www.datadoghq.com/blog/fluentbit-integration-announcement/"
+  tag: "Blog"
+  text: "Centralize your logs with Datadog and Fluent Bit"
+---
+
+## Overview
+
+Configure Fluent Bit to collect, parse, and forward log data from several different sources to Datadog for monitoring. Fluent Bit has a small memory footprint (~450 KB), so you can use it to collect logs in environments with limited resources, such as containerized services and embedded Linux systems. [Datadog's Fluent Bit output plugin][2] supports Fluent Bit v1.3.0+.
+
+## Setup
+### Log collection
+
+Before you begin, you need to have Fluent Bit [installed][6] and [configured][7]. You also need a [Datadog account][3], a [Datadog API key][4], and you need to [activate Datadog Logs Management][5]. These steps assume that you are managing Fluent Bit using their reccomended method of a configuration file, and not through the command line. 
+
+1. Update your [Fluent Bit configuration file][8] to add Datadog as an output plugin. For more information on the configuration parameters, see the [Configuration parameters table](#configuration-parameters). For an example `[OUTPUT]` configuration section, see the [Configuration file example](#configuration-file-example)
+2. [Recompile your Fluent Bit build][9].
```

Comment:
I haven't tried it myself, but I'm not sure that these are the right steps. Ideally, the configuration below will not require recompiling the library. Can we double check this?